hugo/content/about.md

```diff
@@ -0,0 +1,58 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.
+
+## Source Code & Development
+
+The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see [Overview of Repositories on GitHub](https://github.com/mumble-voip/).
+
+## Licenses
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a BSD-style [License](https://www.mumble.info/LICENSE).
```

Comment:
Suggestion: We already link FOSS above. I would drop it here.

IMO linking the word _License_ here does not read fluent. It would be better, but still not optimal, to link _a BSD-style License_. I would add another sentence to signify that we are linking to the LICENSE/LICENSE file specifically here.

```suggestion
Mumble is Free and Open-Source Software and licensed under [a BSD-style License](https://www.mumble.info/LICENSE).
```

```suggestion
Mumble is Free and Open-Source Software and licensed under a BSD-style License. See our [LICENSE file](https://www.mumble.info/LICENSE) for details.
```

Side note: We should probably mention our varying used licenses here, not just the project source code, and provide a reasonable intro to that. But this is fine for a starter.

Reply:
> IMO linking the word License here does not read fluent. It would be better, but still not optimal, to link a BSD-style License. I would add another sentence to signify that we are linking to the LICENSE/LICENSE file specifically here.

I agree, so I will use your second suggestion, correct?

> Side note: We should probably mention our varying used licenses here, not just the project source code, and provide a reasonable intro to that. But this is fine for a starter.

It is mentioned below, but it can be improved I guess.

Reply:
Yes